src/components/TradeWidget/utils.ts

```diff
@@ -16,7 +17,11 @@ export function calculateReceiveAmount(priceValue: string, sellValue: string): s
 
     if (sellAmount && price) {
       const receiveBigNumber = sellAmount.dividedBy(price)
-      receiveAmount = receiveBigNumber.isNaN() || !receiveBigNumber.isFinite() ? '0' : receiveBigNumber.toString(10)
+      // Receive at least input value
+      // uses amount floor < 0.0001
+      // but Limit Price set shows full amount
```

Comment:
I get the change below, but I didn't understand the comment

Reply:
changed it